apps/src/templates/ContentContainer.jsx

```diff
@@ -66,14 +66,14 @@ const styles = {
     float: 'left',
     paddingLeft: 10,
   },
+  children: {
```

Comment:
does this look good with only one item? 
did you take a look at the project widget too?